docs/consultation/SVD-WG011.md

```diff
@@ -0,0 +1,141 @@
+# Nomenclature of positions upstream and downstream of transcripts
+
+## Introduction
+In the meeting of 7-10-2024, the committee decided that positions upstream and
+downstream of a transcript may be addressed using the `c.` and `n.` coordinate
+systems as long as these positions are within the boundaries of the reference
+sequence. A notable advantage of this is the ability to address promoter and
+regulatory regions relative to the transcript they act upon. The committee did,
+however, not decide on the syntax used to address these positions.
+
+## Considerations
+The project leaders of the Variant Validator and Mutalyzer have discussed this
+issue briefly and both agree that more clarification is needed before they can
+proceed further. They have identified three candidates, a fourth was added
+later:
+
+1. Exonic numbering, i.e., relative to the start or end of the coding region.
```

Comment:
Instead of "exonic numbering", how about "CDS numbering" or "CDS coordinates"? "CDS" is consistent with the c. and seems more precise: exonic numbering could mean numbering the exons themselves, or perhaps even numbering relative to the start of the exon.

Reply:
I misunderstood the naming here. Withdrawing comment and resolving.